src/static/js/routes/common.js

```diff
@@ -31,6 +29,7 @@ $( document ).ready( function() { // eslint-disable-line max-statements, no-inli
   require( '../modules/show-hide-fields.js' ).init();
   require( '../modules/external-site.js' ).init();
   require( '../modules/external-site-redirect.js' ).init();
+  require( '../modules/nav-primary.js' ).init();
```

Comment:
It's loaded in order. The reason to have a `init` function is so the module require and initialization can be decoupled if needed.